src/Network/MPD/Applicative/PlaybackControl.hs

```diff
@@ -57,6 +58,14 @@ seek pos time = Command emptyResponse ["seek" <@> pos <++> time]
 seekId :: Id -> FractionalSeconds -> Command ()
 seekId id' time = Command emptyResponse ["seekid" <@> id' <++> time]
 
+-- | Seek to time in the current song. Absolute time for True in
+-- the frist argument, relative time for False.
+-- @since 0.9.2.0
```

Comment:
@iyefrat Question before merging: Did you check if this actually renders properly in haddock ? IIRC, last time I did like this it didn't work. I had to space before and also after (although I'm not sure if the after part is required) the annotation for it to work i.e something like this:
```
--
-- @since 0.9.2.0
--
```

Reply:
You're right sorry about that, I added the empty line and now it works (with haddock 23.1 at least). The part after is apparently not needed.